dora/shell/src/main/java/alluxio/cli/fsadmin/report/SummaryOutput.java

```diff
@@ -0,0 +1,258 @@
+package alluxio.cli.fsadmin.report;
+
+import alluxio.cli.fsadmin.FileSystemAdminShellUtils;
+import alluxio.grpc.MasterInfo;
+import alluxio.grpc.MasterVersion;
+import alluxio.wire.BlockMasterInfo;
+import alluxio.util.CommonUtils;
+import alluxio.util.FormatUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class SummaryOutput {
+    private String mMasterAddress;
+    private int mWebPort;
+    private int mRpcPort;
+    private String mStarted;
+    private String mUptime;
+    private String mVersion;
+    private boolean mSafeMode;
+
+    private List<String> mZookeeperAddress;
+    private boolean mZookeeper;
+    private List<String> mRaftJournalAddress;
+    private boolean mRaftJournal;
+    private List<SerializableMasterVersion> mMasterVersions;
+
+    private int mLiveWorkers;
+    private int mLostWorkers;
+    private Map<String, String> mTotalCapacityOnTiers;
+    private Map<String, String> mUsedCapacityOnTiers;
+    private String mFreeCapacity;
+
+    private static class SerializableMasterVersion {
+        private String mHost;
+        private int mPort;
+        private String mState;
+        private String mVersion;
+
+        public SerializableMasterVersion(MasterVersion masterVersion) {
+            mHost = masterVersion.getAddresses().getHost();
+            mPort = masterVersion.getAddresses().getRpcPort();
+            mState = masterVersion.getState();
+            mVersion = masterVersion.getVersion();
+        }
+
+        public String getHost() {
+            return mHost;
+        }
+
+        public void setHost(String host) {
+            mHost = host;
+        }
+
+        public int getPort() {
+            return mPort;
+        }
+
+        public void setPort(int port) {
+            mPort = port;
+        }
+
+        public String getState() {
+            return mState;
+        }
+
+        public void setState(String state) {
+            mState = state;
+        }
+
+        public String getVersion() {
+            return mVersion;
+        }
+
+        public void setVersion(String version) {
+            mVersion = version;
+        }
+    }
+
+    public SummaryOutput(MasterInfo masterInfo, BlockMasterInfo blockMasterInfo, String dateFormatPattern) {
+        // give values to internal properties
+        mMasterAddress = masterInfo.getLeaderMasterAddress();
+        mWebPort = masterInfo.getWebPort();
```

Comment:
the web port and raft journal port are all here, are they displayed too?

Reply:
As to the content to display, I followed the original format